app/controllers/ratings_controller.rb

```diff
@@ -1,11 +1,10 @@
 class RatingsController < ApplicationController
 
-
   def rate_meals
     params[:ratings].each do |rating_params|
       if rating_params[:value].present?
         @meal = Meal.find_by(id: rating_params[:meal_id])
```

Comment:
what if meal is not found? First check if meal is loaded.

Reply:
discussed